Metrics/collect_and_upload.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+xcov --workspace Example/Firebase.xcworkspace --scheme AllUnitTests_iOS --output_directory Metrics --json_report
+pushd Metrics
```

Comment:
`pushd` adds extra noise to the output. Since this script doesn't need to run the closing `popd`, you can just use `cd` instead which is quieter.